src/main/java/uk/nhs/digital/nhsconnect/lab/results/configuration/AmqpConfiguration.java

```diff
@@ -0,0 +1,102 @@
+package uk.nhs.digital.nhsconnect.lab.results.configuration;
+
+
+import org.apache.qpid.jms.JmsConnectionFactory;
+import org.apache.qpid.jms.JmsDestination;
+import org.apache.qpid.jms.message.JmsMessageSupport;
+import org.apache.qpid.jms.policy.JmsDefaultDeserializationPolicy;
+import org.apache.qpid.jms.policy.JmsRedeliveryPolicy;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jms.support.converter.MappingJackson2MessageConverter;
+import org.springframework.jms.support.converter.MessageConverter;
+import org.springframework.util.StringUtils;
```

Comment:
This relates to another comment I made below about using Apache Common's `StringUtils.isBlank` method over Spring's `StringUtils.isEmpty`

```suggestion
import org.apache.commons.lang3.StringUtils;
```